db-drda/src/main/java/com/splicemachine/db/impl/drda/DRDAConnThread.java

```diff
@@ -6508,6 +6508,25 @@ private String buildTokenizedSqlerrmc(SQLException se) {
 		do {
 			if ( se instanceof EmbedSQLException)
 			{
+				StringBuilder sb = new StringBuilder();
```

Comment:
Why do we need special handling here for Signaled exceptions?

Reply:
Because for StandardException Splice looks up the error message text based on the error code, so if it's not a predefined error code, nothing is found.  Here's the result with the above code removed:
```
splice> signal sqlstate '12345' ('diagnostic message');
ERROR 12345: 12345 : [0] 
```